### ISSUE TITLE
[Bug] UWP ListView ContextActions menu not accessible using keyboard

### ISSUE THREAD

**Issue by johnshardman** [xamarin/Xamarin.Forms#12355]
### Description

It appears that the XF ListView does not support keyboard access to the ContextActions menu on UWP, making it inaccessible to many users who rely on the keyboard or accessibility tools that simulate the keyboard.

### Steps to Reproduce

Run any existing ContextActions sample and try to pop up the context menu for a ListView item using just the keyboard on UWP.

### Expected Behavior

The correct key combination should result in the context menu appearing and the focus being put on the first menu item in that context menu.

### Actual Behavior

No key combination tried results in the context menu appearing.

### Basic Information

- Version with issue: XF 4.8

### Workaround

No workaround found. Looking at the source code (ok, I checked against an early snapshot of the Maui repository, but I doubt things have changed in the corresponding code in XF recently), pretty much everything in CellControl (such as OpenContextMenu) is neither public nor protected, so painful for a custom renderer to get to (it would currently require reflection to get to internal methods etc).

**Comment by schovan** [comment]
In a UWP application on Xbox Series S, pressing Menu Button on Xbox controller doesn't pop up the context menu for a selected ListView item.

**Comment by jfversluis** [comment]
Let's consider this for .NET MAUI, thanks for the suggestion!
